athenahealth/httpclient.go

```diff
@@ -178,15 +175,10 @@ func (h *HTTPClient) request(ctx context.Context, method, path string, body io.R
 	req.Header.Add("User-Agent", userAgent)
 	req.Header.Set(XRequestIDHeaderKey, xRequestID)
 
-	var requestBodyLength int64
-	if srBody, ok := body.(*sizeRecordingReader); ok {
-		requestBodyLength = srBody.size
-	}
-
 	h.logger.Info().
 		Str("method", method).
 		Str("url", reqURL).
-		Int64("bodyLength", requestBodyLength).
+		Int64("bodyLength", req.ContentLength).
```

Comment:
Could we retain the `sizeRecordingReader` and add a reqBodySize field into the log after `httpClient.Do()` in the `athenahealth API response` log below?